opm/common/utility/numeric/GridUtil.cpp

```diff
@@ -0,0 +1,108 @@
+#include <opm/common/utility/numeric/GridUtil.hpp>
+#include <tuple>
+#include <iostream>
```

Comment:
I don't think you need `<iostream>` here.  If you need to output something to the user, we should generally prefer the logging system over "bare" iostream-based output.  If this is leftover from initial development then the header should just be removed.